docs/notebooks/text_to_image.ipynb

```diff
@@ -0,0 +1,367 @@
+        "\n",
+        "clip_text_encoder = finetuner.get_model(artifact=artifact, select_model='clip-text')\n",
+        "clip_image_encoder = finetuner.get_model(artifact=artifact, select_model='clip-vision')\n",
+        "\n",
+        "finetuner.encode(model=clip_text_encoder, data=text_da)\n",
+        "finetuner.encode(model=clip_image_encoder, data=image_da)\n",
+        "\n",
+        "print(text_da.embeddings.shape)\n",
+        "print(image_da.embeddings.shape)"
+      ],
+      "metadata": {
+        "id": "v95QsuEyzE-B"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "```bash\n",
+        "(1, 512)\n",
+        "(1, 512)\n",
+        "```\n",
+        "\n",
+        "```{admonition} what is select_model?\n",
+        "When fine-tuning CLIP, we are fine-tuning the CLIPVisionEncoder and CLIPTextEncoder in parallel.\n",
+        "The artifact contains two models: `clip-vision` and `clip-text`.\n",
+        "The parameter `select_model` tells finetuner which model to use for inference, in the above example,\n",
+        "we use `clip-text` to encode a Document with text content.\n",
+        "```\n",
+        "\n",
+        "```{admonition} Inference with ONNX\n",
+        "In case you set `to_onnx=True` when calling `finetuner.fit` function,\n",
+        "please use `model = finetuner.get_model('/path/to/YOUR-MODEL.zip', is_onnx=True)`\n",
+        "```"
+      ],
+      "metadata": {
+        "id": "LzMbR7VgzXtA"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Advanced: WiSE-FT \n",
+        "\n",
+        "WiSE-FT, proposed by Mitchell et al. in [Robust fine-tuning of zero-shot models](https://arxiv.org/abs/2109.01903),\n",
+        "has been proven to be an effective way for fine-tuning models with a strong zero-shot capability,\n",
+        "such as CLIP.\n",
+        "As was introduced in the paper:\n",
+        "\n",
+        "> Large pre-trained models such as CLIP or ALIGN offer consistent accuracy across a range of data distributions when performing zero-shot inference (i.e., without fine-tuning on a specific dataset). Although existing fine-tuning methods substantially improve accuracy on a given target distribution, they often reduce robustness to distribution shifts. We address this tension by introducing a simple and effective method for improving robustness while fine-tuning: ensembling the weights of the zero-shot and fine-tuned models (WiSE-FT).\n",
+        "\n",
+        "Finetuner allows you to apply WiSE-FT easily,\n",
+        "all you need to do is use the `WiSEFTCallback`.\n",
+        "Finetuner will trigger the callback when fine-tuning job finished and merge the weights between the pre-trained model and the fine-tuned model:\n",
+        "\n",
+        "```diff\n",
+        "from finetuner.callbakcs import WiSEFTCallback\n",
```

Comment:
This import is wrong and has to be fixed it is
`from finetuner.callback import WiSEFTCallback`